components/dropdown/dropdown.jsx

```diff
@@ -12,11 +12,13 @@ export default class Dropdown extends React.Component {
 
 		return (
 			<div
+                                tabIndex="0"
```

Comment:
It looks like maybe your editor wasn't listening to the editorconfig on this one, can you fix the formatting?